src/VisualStudio/Core/Def/Implementation/FindReferences/StreamingFindUsagesPresenter.cs

```diff
@@ -170,8 +168,9 @@ private AbstractTableDataSourceFindUsagesContext StartSearchWorker(string title,
         {
             this.AssertIsForeground();
```

Comment:
so starting the search is guaranteed to happen on the UI thread.  So why not just have the person who is about to start the search just create the presenter on hte UI thread as well (that was always the intent).